Tasks/PublishBuildArtifacts/task.json

```diff
@@ -21,15 +21,15 @@
       "name": "PathtoPublish",
       "type": "filePath",
       "label": "Path to publish",
-      "defaultValue": "",
+      "defaultValue": "$(Build.ArtifactStagingDirectory)",
       "required": true,
       "helpMarkDown": "The folder or file path to publish. This can be a fully-qualified path or a path relative to the root of the repository. Wildcards are not supported. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) are supported. Example: $(Build.ArtifactStagingDirectory)"
     },
     {
       "name": "ArtifactName",
       "type": "string",
       "label": "Artifact name",
-      "defaultValue": "",
+      "defaultValue": "artifact",
```

Comment:
In the designer templates I spot checked, the artifact name is defaulted to "drop". Can we standardize on that?

Reply:
@hashtagchris thanks for checking that!  Yes, let's use the same name in both places.  Somehow, "drop" is vague to me.  Internally we refer to "build drops", but is that old school, and what's common elsewhere?  Since they're called "build artifacts," I thought "artifact" would be better.  But maybe I'm wrong.  Is "drop" more meaningful to people?

Reply:
I think "drop" was used to signify build drop. We should check if changing this will have an undesirable impact on the UI or elsewhere

Reply:
If we're going to change the default, what do you think of **"build"**? App Center has this:

![image](https://user-images.githubusercontent.com/7145717/39409598-d63b86c0-4bb7-11e8-8702-cd0dd1c03d5d.png)

I think having the default name **"artifact"** may make it harder to communicate that there are multiple artifact categories and that you can publish multiple build artifacts. Here's what you would get with our .Net Desktop template:

![image](https://user-images.githubusercontent.com/7145717/39409548-162a85d4-4bb7-11e8-9783-e7b9cdfefaf0.png)

Also the artifact name is used in file paths for unc shares and downloads, and 8 characters puts us closer to the 260 character file limit. Admittedly having "build" in those file paths may be a bit odd: \\myshare\builds\20180401.01\build\Release\ConsoleApp1.exe

Reply:
Agree "artifact" seems overloaded and will create confusion. "drop" is a fairly common term to indicate a build output.